src/App/integrationTests/App.useSyncApiDataIntoOfflineStorage.test.js

```diff
@@ -153,10 +153,10 @@ test('Sync: initial page load already done, navigate to non project page', async
   /**
    * because of the other landing page initial load sync test,
    * we can rely on that data already being here and focus on the
-   * the data that hasn't yet been synced. A project page nav sync
+   * the data that has not yet been synced. A project page nav sync
    * will technically pull more than this
    * (the stuff that gets pulled on a non project page load minus choices),
-   * but we are going to just test the data that hasn't already been pulled
+   * but we are going to just test the data that has not already been pulled
```

Comment:
have not

Reply:
This is in a comment. It got changed because it was in a find-and-replace in a test file.